src/index.ml

```diff
@@ -345,7 +345,7 @@ struct
             l "[%s] log file detected. Loading %Ld entries"
               (Filename.basename root) entries);
         let mem = Tbl.create (Int64.to_int entries) in
-        iter_io (fun e -> Tbl.replace mem e.key e.value) io;
+        iter_io (fun e -> Tbl.add mem e.key e.value) io;
```

Comment:
This is RW code so it will ultimately lead to duplicates in the index file.

Reply:
But it is loading entries from the log file, which shouldn't contain duplicates, no?

Reply:
Yes, you are right. Thanks !

I changed it back and also added two tests for duplicate entries, (one that fails if add is used here). In the test I noticed that `iter` calls iter on index twice, so I remove one of them.